Example/Podfile

```diff
@@ -13,7 +13,7 @@ target 'ExampleApp' do
   use_frameworks!
 
   # Pods for ExampleApp
-  pod 'SalemoveSDK', :git => 'https://github.com/salemove/ios-bundle', :tag => "0.29.6"
+  pod 'SalemoveSDK', :git => 'https://github.com/salemove/ios-bundle', :tag => "0.29.7"
```

Comment:
I believe that doing this change will make Bitrise fail because it will try to find a pod with that tag, which doesn't exist yet. That is why once we release, we need to make a PR in ios-bundle to update it. I can be wrong though, let's see what happens.